src/sumo/wrapper/login.py

```diff
@@ -55,6 +56,15 @@ def get_parser() -> ArgumentParser:
         help="Print access token",
     )
 
+    parser.add_argument(
+        "-s",
+        "--silent",
+        dest="silent",
+        action="store_true",
+        default=False,
```

Comment:
I _think_ it is unnecessary to default to `False` here as the "store_true" will automatically default to `False`

Reply:
True, unnecessary. Could argue that adding `default=False` increases readability(?)